### ISSUE TITLE
Improve Embedding results by finetuning

### ISSUE THREAD

**Issue by dlangenk** [biigle/maia#141]
We could finetune the pretrained weights to be more adapted to the marine domain (at the loss of other domains in Biigle, I guess). This can also be done in a self-supervised fashion and might also be a valid fit for our new PhD candidate.

**Comment by mzur** [comment]
Anything can be tried once we have a finetuned model. Currently, I'm putting it more into the MAIA/detection category which for me is still something special for the marine domain. The general similarity sorting should work well in any domain and I'm not sure if we should employ such a fine-tuned model there. But the numbers will tell :wink:
